src/test/scala/com/cognite/spark/datasource/BaseUrlTest.scala

```diff
@@ -0,0 +1,18 @@
+package com.cognite.spark.datasource
+
+import org.scalatest.FlatSpec
+
+class BaseUrlTest extends FlatSpec with SparkTest {
+  private val greenfieldApiKey = System.getenv("TEST_API_KEY_GREENFIELD")
+
+  it should "list the correct number of 3D models on greenfield.cognitedata.com" taggedAs GreenfieldTest in {
+    val df = spark.read
+      .format("com.cognite.spark.datasource")
+      .option("apiKey", greenfieldApiKey)
+      .option("type", "3dmodels")
+      .option("limit", 4)
+      .option("baseUrl", "https://greenfield.cognitedata.com")
```

Comment:
This test wouldn't catch if this parameter was ignored. Can we find something where we can load from each project and compare the results, making sure they're different? Maybe files metadata is a good option, just counting the number of files in each and ensuring they're not the same?